GraphView/graphview.py

```diff
@@ -2377,6 +2377,12 @@ def get_tags_and_table(self, obj):
             tag_table = ('<TABLE BORDER="0" CELLBORDER="0" '
                          'CELLPADDING="5"><TR>')
             for tag in tags:
+                rgba = Gdk.RGBA()
+                rgba.parse(tag.get_color())
+                value = '#%02x%02x%02x' % (int(rgba.red * 255),
+                                   int(rgba.green * 255),
+                                   int(rgba.blue * 255))
+                tag.set_color(value)
```

Comment:
Should we really need to set color to tag?
`tag.set_color(value)`
Maybe just use generated value?
`tag_table += '<TD BGCOLOR="%s"></TD>' % value`

Reply:
I do not know ... Does it work too?
In my mind, it was for re-assigning
a value after the conversion, 
like done by the hand via GUI.

Yes, feel free. Thanks.

Reply:
Note, as there is no DB commit,
I only supposed that [setting color](https://www.gramps-project.org/docs/gen/gen_lib.html?highlight=tag#gramps.gen.lib.tag.Tag.set_co) 
was in memory.

Reply:
Ok for me as we don't commiting to db.